### PR TITLE
Update components HTML snippet to match nunjucks templates

### PR DESCRIPTION
**Pull request by AdamChrimes**
## Description

- Update installing with compiled documentation page template with skip link, header and footer components.
- Remove white space and empty lines from nunjucks templates and make sure that each component HTML snippets match the nunjuck example.
- Make sure that each of the HTML snippets work using compiled files rather than npm package.
- Header HTML snippet was missing the new autocomplete `<div>`
- Pagination HTML snippet was using the old class name for the visually hidden utility class
- Add missing closing `</div>` to the promo HTML snippet.

### CODE REVIEW

File: packages/components/promo/README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ The small promo Nunjucks macro takes the following arguments:
   </div>
   <div class="nhsuk-grid-column-one-third nhsuk-promo-group__item">
     <div class="nhsuk-promo">
-      <a class="nhsuk-promo__link-wrapper" href="https://www.nhs.uk">
+      <a class="nhsuk-promo__link-wrapper"  href="https://www.nhs.uk">
```
Comment (mcheung-nhs) [inline packages/components/promo/README.md]: Remove extra space?

File: packages/components/promo/README.md [hunk 2]
```diff
@@ -202,7 +202,7 @@ The small promo Nunjucks macro takes the following arguments:
...
```
Comment (mcheung-nhs) [inline packages/components/promo/README.md]: Remove extra space?

File: packages/components/radios/README.md [hunk 3]
```diff
@@ -75,36 +75,28 @@ To discuss or contribute to this component, visit the [GitHub issue for this com
...
 <div class="nhsuk-form-group">
-
-  <fieldset class="nhsuk-fieldset" aria-describedby="example-hint">
-
-  <legend class="nhsuk-fieldset__legend">
-    Have you changed your name?
-  </legend>
-
-  <span class="nhsuk-hint" id="example-hint">
+  <fieldset class="nhsuk-fieldset" aria-describedby="example&#39;-hint">
```
Comment (mcheung-nhs) [inline packages/components/radios/README.md]: Dodgy character - think this was fixed here?
https://github.com/nhsuk/nhsuk-frontend/pull/354

File: packages/components/radios/README.md [hunk 4]
```diff
@@ -75,36 +75,28 @@ To discuss or contribute to this component, visit the [GitHub issue for this com
...
-  <div class="nhsuk-radios nhsuk-radios--inline">
-
-    <div class="nhsuk-radios__item">
-      <input class="nhsuk-radios__input" id="example-1" name="example" type="radio" value="yes">
-      <label class="nhsuk-label nhsuk-radios__label" for="example-1">
+    </span>
+    <div class="nhsuk-radios nhsuk-radios--inline">
+      <div class="nhsuk-radios__item">
+        <input class="nhsuk-radios__input" id="example&#39;-1" name="example"type="radio" value="yes">
+        <label class="nhsuk-label nhsuk-radios__label" for="example&#39;-1">
```
Comment (mcheung-nhs) [inline packages/components/radios/README.md]: Dodgy character - see above

File: packages/components/radios/README.md [hunk 5]
```diff
@@ -75,36 +75,28 @@ To discuss or contribute to this component, visit the [GitHub issue for this com
...
-    </div>
-
-    <div class="nhsuk-radios__item">
-      <input class="nhsuk-radios__input" id="example-2" name="example" type="radio" value="no" checked>
-      <label class="nhsuk-label nhsuk-radios__label" for="example-2">
+        </label>
+      </div>
+      <div class="nhsuk-radios__item">
+        <input class="nhsuk-radios__input" id="example&#39;-2" name="example"type="radio" value="no" checked>
+        <label class="nhsuk-label nhsuk-radios__label" for="example&#39;-2">
```
Comment (mcheung-nhs) [inline packages/components/radios/README.md]: Dodgy character - see above

File: app/components/radios/inline.njk [hunk 6]
```diff
@@ -11,7 +11,7 @@
       <div class="nhsuk-grid-row">
         <div class="nhsuk-grid-column-two-thirds">
           {{ radios({
-            "idPrefix": "example'",
```
Comment (AdamChrimes) [inline app/components/radios/inline.njk]: Pesky random `'` @mcheung-nhs

File: packages/components/checkboxes/README.md [hunk 7]
```diff
@@ -157,32 +157,26 @@ To discuss or contribute to this component, visit the [GitHub issue for this com
...
       <input class="nhsuk-checkboxes__input" id="colours-2" name="colours" type="checkbox" value="green">
       <label class="nhsuk-label nhsuk-checkboxes__label" for="colours-2">
-        Green
+      Green
       </label>
     </div>
-
     <div class="nhsuk-checkboxes__item">
-      <input class="nhsuk-checkboxes__input" id="colours-3" name="colours" type="checkbox" value="blue" disabled>
+      <input class="nhsuk-checkboxes__input" id="colours-3" name="colours" type="checkbox" value="blue" disabled="">
```
Comment (mcheung-nhs) [inline packages/components/checkboxes/README.md]: Is `=""` required for the disabled attribute?

**Pull request merged**